### PR TITLE
Donors: Include total donations in list.

### PR DESCRIPTION
**Pull request by xim**
This fixes issue #463

Note that this requires that stiftelsen-effekt/effekt-backend#463 be merged

### CODE REVIEW

File: src/components/pages/donors/Donor.tsx [hunk 1]
```diff
@@ -58,8 +58,11 @@ export const DonorPage: React.FunctionComponent<RouteComponentProps<IParams>> =
   let totalDonations = 0;
   let operationsDonations = 0;
   if (data.stats && data.stats.sumYearlyAggregates) {
+    console.log(data.stats.sumYearlyAggregates);
```
Comment (fellmirr) [inline src/components/pages/donors/Donor.tsx:61]: Remove console log

File: src/components/pages/donors/Donor.tsx [hunk 2]
```diff
@@ -58,8 +58,11 @@ export const DonorPage: React.FunctionComponent<RouteComponentProps<IParams>> =
...
     data.stats.sumYearlyAggregates.forEach((row) => {
       let amount = row.value ? row.value.toNumber() : 0;
+      if (amount)
```
Comment (fellmirr) [inline src/components/pages/donors/Donor.tsx:64]: Remove console log

**Review by fellmirr — CHANGES_REQUESTED** [review]

**Comment by xim** [comment]
Embarrasing `>_<`

**Review by fellmirr — APPROVED** [review]
Looks good 👍

**Pull request merged**